tests/integration/golang/aim/app/get_app_test.go

```diff
@@ -0,0 +1,88 @@
+//go:build integration
+package run
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetAppTestSuite struct {
+	suite.Suite
+	client      *helpers.HttpClient
+	appFixtures *fixtures.AppFixtures
+	apps        []*models.App
+}
+
+func TestGetAppTestSuite(t *testing.T) {
+	suite.Run(t, new(GetAppTestSuite))
+}
+
+func (s *GetAppTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(os.Getenv("SERVICE_BASE_URL"))
+
+	appFixtures, err := fixtures.NewAppFixtures(os.Getenv("DATABASE_DSN"))
+	assert.Nil(s.T(), err)
+	s.appFixtures = appFixtures
+
+	s.apps, err = s.appFixtures.CreateTestApps(context.Background(), 10)
```

Comment:
we don't need to create 10 app here. this is a bit misleading. can you please create just one.